### PR TITLE
Created a test file for each component with tests or comments with tests to write

### PR DESCRIPTION
**Pull request by macrawford**

### CODE REVIEW

File: client/src/tests/questions.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow, mount, render } from 'enzyme';
+import Questions from '../Components/Questions.jsx';
+
+// These tests are not passing, I may have to pass in mock data for the questions props?
```
Comment (jnweiner) [inline client/src/tests/questions.test.js:6]: hmm, is it giving you a specific error?

Comment (macrawford) [reply client/src/tests/questions.test.js:6]: I think it has to do with the fact that my component maps over some props. If I don't pass in any real data I don't think it runs...

<Questions /> › it should call a function when the next page button is clicked

    TypeError: this.props.questions.map is not a function

      218 |     return (
      219 |       <Body className='questionModule'>
    > 220 |         {this.props.questions.map((question, questionIndex) => {
          |                               ^
      221 |           if (questionIndex >= this.state.questionMin && questionIndex < this.state.questionMax) return (
      222 |             <QuestionModule key={questionIndex} className='question'>
      223 |               <Question>

File: client/src/tests/questions.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+describe('<Questions />', () => {
+  test('it should call a function when the previous page button is clicked', () => {
+    const mockSubmitHandler = jest.fn();
+    const wrapper = shallow(<Questions questions={{}}/>);
+    wrapper.instance().nextPage = mockSubmitHandler;
```
Comment (jnweiner) [inline client/src/tests/questions.test.js:12]: looks like you may have the nextPage and prevPage properties switched - this test is referencing nextPage, and the following test is referencing prevPage

**Comment by jnweiner** [comment]
looking great!! just left a thought about the tests that are failing

**Pull request merged**